src/foam/u2/filter/FilterConfigView.js

```diff
@@ -50,7 +50,10 @@ foam.CLASS({
     {
       class: 'String',
       name: 'filterSearch',
-      placeholder: 'Search...'
+      view: {
+        class: 'foam.u2.SearchField',
+        onKey: true
+      }
```

Comment:
you can also just set onKey on the property, the view picks it up from the property